src/components/Header/Header.stories.tsx

```diff
@@ -9,6 +9,8 @@ import { TagList } from '../TagList/TagList';
 import { Header } from './Header';
 import { MOCK_HEADER_PROPS, MOCK_HEADER_PROPS_FULL } from './Header.mock';
 import { HeaderAvatar, HeaderButtons, HeaderTags } from './Header.slots';
+import { ButtonToolbar } from '..';
```

Comment:
Dit lijkt volgens mij mogelijks circular dependencies te veroorzaken. Ik zou importen van de ButtonToolbar component zelf.

Reply:
yep, mijn editor die denkt dat ie slim is